src/AutoComplete/AutoComplete.js

```diff
@@ -266,6 +266,9 @@ class AutoComplete extends Component {
   };
 
   handleItemTouchTap = (event, child) => {
+    // Prevent ghost clicks
+    event.preventDefault();
```

Comment:
What about exposing a property so it can be implemented on user-land. I'm worried about the side effects here?